docs/interfaces/hmx/start_signal.md

```diff
@@ -6,11 +6,13 @@ tags:
 ---
 
 ## Overview
-There are 5 start signals on different connectors. Start 1 to 4 is meant to be used to start the device and Start 5 is meant to be used with an external LED or an output for signal start.
+There are 5 start signals on different connectors. Start 1 to 4 is meant to be used to start the device and Start 5 is meant to be used with an external LED or an output for signal start. Only one external start signal is needed to start the device. They are all connected to a a singel input and it can be read from gpio-key event or unbind gpio-key to use this signal and other digital in signals as regular gpios.
```

Comment:
"a a singel" -> "a single"

Reply:
```suggestion
There are 5 start signals on different connectors. Start 1 to 4 is meant to be used to start the device and Start 5 is meant to be used with an external LED or an output for signal start. Only one external start signal is needed to start the device. They are all connected to a single input and it can be read from gpio-key event or unbind gpio-key to use this signal and other digital in signals as regular gpios.
```